usecase/usecase-webapi-xwalk-tests/samples/IapGooglePlay/index.html

```diff
@@ -26,7 +26,7 @@
 EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 Authors:
-        Liu,Yun <yunx.liu@intel.com>
+        Zhong, Qiu <zhongx.qiu@intel.com>
```

Comment:
Please don't remove existing authors. Please use `Qiu, Zhong` instead of `Zhong, Qiu`. You can update `Liu, Yun`, a space after `,`.